### DISCUSSION TITLE
How to request to catch up new release of vte3

### DISCUSSION THREAD

**Discussion by inoueta** [Homebrew/discussions#544] [Everyday usage]
https://gitlab.gnome.org/gnome/vte/-/archive/0.62.2/vte-0.62.2.tar

Hi all, I have one question.
There is a new release of vte3 source from gnome team at

https://gitlab.gnome.org/gnome/vte/-/archive/0.62.2/vte-0.62.2.tar

So, I want to request to make the vte3 formula catch up this
because current version does not work on mac due to a bug.
Is brew bump-formula-pr for that purpose?
Best

**Comment by carlocab** [comment]
You can use `brew bump-formula-pr` if you have a GitHub token set up, and if the version bump can be done by a simple substitution of version numbers in the tarball URL. It doesn't look like that's the case here, so you might need to manually create a PR for the version bump.

The docs here should help: https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request

Also, I'm not actually sure that 0.62.2 is a stable release of `vte3`. Only stable releases are acceptable in homebrew/core.

**Reply by inoueta** [reply]
Thank you for the information.  I will try manual PR

Yes. That is stable release.
Just the owner of the software doesn't want to push it to the previous place. I don't know why. Please look at 

https://gitlab.gnome.org/GNOME/vte/-/issues/324

It seems that the primary place of the source was changed to gitlab from this time.
Is it true? Does anyone know better ?

Anyway,  since the vte3 0.62.1 does not work on macOS, we need to update it to 0.62.2.
Please help.

Best
